components/molecules/ContributorHoverCard/contributor-hover-card.tsx

```diff
@@ -31,8 +31,8 @@ const ContributorHoverCard = ({
   isMaintainer,
 }: ContributorHoverCardProps) => {
   const router = useRouter();
-  const { filterName } = router.query;
-  const topic = filterName as string;
+  const { pageId } = router.query;
+  const topic = pageId as string;
```

Comment:
nit: since this is being refactored, there is no need to destructure as it's only used once.

There's other places in the other files that have the same changes.

```suggestion
  const topic = router.query.pageId as string;
```

Reply:
Thanks for the review, Nick. I've done the refactor. Please, feel free to have a look when you've time.